tests/grate/asm/fs_vs_tests.txt

```diff
@@ -0,0 +1,13 @@
+alu_buffer_size = 1
+pseq_to_dw_exec_nb = 1
+
+.asm
+EXEC
+	MFU:	var unk(0x104E51BA0) r0.fp20, r0.fp20, r0.fp20, r0.fp20
```

Comment:
Yeah, it should contain barycentric weights. I don't think it's instructions by itself, but a part of the varying (load from TRAM and interpolate) operation. If they are always constant, we may derive it into a some modifier literal, like (setup) instead of unk(..), for convenience.